### PR TITLE
[testdriver] Add `test_driver.bidi.permissions.set_permission`

### PR DESCRIPTION
**Pull request by sadym-chromium**
Add [`permissions.setPermission`](https://w3c.github.io/permissions/#webdriver-bidi-command-permissions-setPermission) command to `test_driver.bidi.permissions.set_permission`.

RFC is not required, as the method matches a WebDriver BiDi command.

### CODE REVIEW

File: infrastructure/metadata/infrastructure/webdriver/bidi/set_permission.https.html.ini [hunk 1]
```diff
@@ -0,0 +1 @@
+disabled: https://github.com/web-platform-tests/wpt/issues/47544
```
Comment (OrKoN) [inline infrastructure/metadata/infrastructure/webdriver/bidi/set_permission.https.html.ini:1]: Looks like there is a test failure for this test https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=130038&view=logs&jobId=af75b4de-b26f-5d4d-58e8-b41258076d44&j=af75b4de-b26f-5d4d-58e8-b41258076d44&t=2e0f365d-858c-52d3-7da5-72c4b225c9fa so perhaps it does not get disabled properly

**Review by OrKoN — APPROVED** [review]
LGTM

File: infrastructure/metadata/infrastructure/webdriver/bidi/set_permission.https.html.ini (hunk 1 above)
Comment (sadym-chromium) [reply infrastructure/metadata/infrastructure/webdriver/bidi/set_permission.https.html.ini]: right, I messed up with the pas. Fixed now.

**Comment by sadym-chromium** [comment]
@OrKoN should the external specifications be stored in **external** `test_driver.bidi.external.permissions.set_permission`?

**Comment by OrKoN** [comment]
> @OrKoN should the external specifications be stored in **external** `test_driver.bidi.external.permissions.set_permission`?

I believe the external thing is specific to organizing tests of WebDriver BiDi in WPT. The classic permission spec was also external and it was not prefixed with external in neither classic tests tests nor in testdriver.js API. So, no, I do not think we need .external in testdriver.js.

**Review by past — APPROVED** [review]

**Pull request merged**